packages/koa-shopify-webhooks/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 <!-- ## [Unreleased] -->
 
+## [2.5.0] - 2020-08-17
```

Comment:
Minor: I take it we don't need to update the `version` in `package.json` since it's never been updated past `1.0.0`? Mostly asking to build my own context on this.

Reply:
I recall reading somewhere in the docs in this repo, that we shouldn't touch `package.json`.